tests/test-integration-mock.py

```diff
@@ -769,6 +769,65 @@ def test_11_preboot_acl_remove_not_present(self):
         # disconnect all devices
         tree.disconnect(self.testbed)
 
+    # Test multi controllers with preboot_acl
+    def test_12_preboot_acl_multi_controller(self):
+        # connect all device
+        # First controller
+        UUID = '00000000-0000-0000-0000-000000000001'
+        device1 = TbDevice("0-2")
```

Comment:
Same here regarding port numbers ("0-3" should be used here)

Reply:
@westeri added patch renaming all tests